helpers/all.js

```diff
@@ -0,0 +1,52 @@
+var _ = require('lodash'),
+    internals = {};
+
+/**
+ * Yield block only if all arguments are valid
+ *
+ * @example
+ * {{#all items theme_settings.optionA theme_settings.optionB}} ... {{/any}}
+ */
+
+internals.implementation = function(handlebars) {
+    this.handlebars = handlebars;
+};
+
+internals.implementation.prototype.register = function(context) {
+    this.handlebars.registerHelper('all', function () {
+
```

Comment:
We typically follow the airbnb style guide. We should work that into the build process as well.